lib/sdf/link.rb

```diff
@@ -19,6 +19,16 @@ def pose
             Conversions.pose_to_eigen(xml.elements["pose"])
         end
 
+        # Check if link is kinematic
+        #
+        #
+        def kinematic
+            if kinematic = xml.elements["kinematic"]
+                return Conversions.to_boolean(kinematic)
```

Comment:
In a simple method like this, the general ruby style is to drop the return. Moreover, `if false; end` is nil.

I.e. would be better written as

```
def kinematic?
  if kinematic = ...
     Conversions.to_boolean(kinematic)
  end
end
```